.gitignore

```diff
@@ -1,10 +1,39 @@
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
+*.o
+*.a
+*.so
+
+# Folders
+_obj
+_test
+
+# Architecture specific extensions/prefixes
+*.[568vq]
```

Comment:
Oh, I haven't the foggiest. This is legacy and can instead be merged with what @kensimon added in #1